### ISSUE TITLE
Dynamic POC баг или не баг?

### ISSUE THREAD

**Issue by 11-12** [SBProX/SBProX-Dev#483]
Уважаемые разработчики, проверьте пожалуйста работу индикатора Dynamic POC.
Обнаружена следующая ситуация: при обновлении графика кнопкой "Применить" индикатор перерисовывает линии РОС и VA и они оказываются на других уровнях,  В большей степени это относится к линиям VA, но при резких движениях цены линия РОС также перерисовывается, если нажать кнопку "Применить"
![Screenshot_57](https://user-images.githubusercontent.com/52030059/62446267-f963e280-b76a-11e9-99cc-5dd59f3e5950.jpg)
![Screenshot_58](https://user-images.githubusercontent.com/52030059/62446273-fe289680-b76a-11e9-8e96-13c8515678b7.jpg)

**Comment by 11-12** [comment]
![1](https://user-images.githubusercontent.com/52030059/62459120-3dfd7700-b787-11e9-876d-dfc2d89f36b2.gif)

**Comment by gios76** [comment]
да.. это проблема действительно существует. Я надеюсь что исрправлят эту ошибку
